server/aws-lsp-codewhisperer/script/install_transitive_dep.js

```diff
@@ -0,0 +1,28 @@
+// This script is required because transitive dependencies of bundled dependencies are not getting packed or installed.
+
+/* eslint-disable import/no-nodejs-modules */
+const fs = require('fs')
+const path = require('path')
+const { execSync } = require('child_process')
+
+const packageJson = JSON.parse(fs.readFileSync('package.json').toString())
+const bundleDependencies = packageJson.bundleDependencies || []
+
+for (let dependency of bundleDependencies) {
+    const dependencyPath = path.join('node_modules', dependency)
+
+    // Check if the dependency exists in node_modules to avoid monorepo setup
+    if (fs.existsSync(dependencyPath)) {
+        console.log(`Bundled dependency '${dependency}' found. Installing its dependencies...`)
+        try {
+            process.chdir(dependencyPath)
+            const npmInstallCommand = process.platform === 'win32' ? 'npm.cmd install' : 'npm install'
```

Comment:
what is the win32 platform? Is it used by any github action environments?

Reply:
I added this to make the package compatible with Windows, otherwise installation of this package would not work